### PR TITLE
Update readme and requirements

### PR DESCRIPTION
**Pull request by apex-omontgomery**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -15,3 +15,5 @@ pycryptodomex
 
 # TODO: Building with a custom openssl 1.1.1 on centos 6 is failing to compile M2Crypto, this should be fixed in the future as M2Crypto is prefered
 #M2Crypto>=0.35.2
+
+pop-build
```
Comment (thatch45) [inline requirements.txt:19]: This requirements.txt file is used to build Salt. pop-build is not used in Salt, so it should not go in this file

File: README.rst [hunk 2]
```diff
@@ -22,26 +22,34 @@ target specific versions of python. This allows us to build Salt with
...
 
-Install pop-build:
+### Clone the repo
+
+* git clone https://github.com/saltstack/salt-bin.git
+* cd salt-bin
+
+### Install dependencies
+
+* pip install -r requirements.txt
```
Comment (thatch45) [inline README.rst:35]: This should just be changed to 
pip install pop-build

As the requirements file is for Salt itself, not the build process

Comment (apex-omontgomery) [reply README.rst:35]: Would you be opposed to having two requirements.txt files
`salt_bin.requirements.txt` for dependencies required to run salt-bin builder
`salt.requirements.txt` for dependencies salt-bin builder uses?

Comment (thatch45) [reply README.rst:35]: I think that the right approach would be a refactor, the requirements should all be in the conf directory so we can distribute salt-bin as a python project.
Let me get this :)

**Pull request closed without merging**